docker-plain/files/README.md

```diff
@@ -0,0 +1,8 @@
+# Plain Docker image (docker-plain)
+
+## How to create a custom jenkins-agent out of this docker-plain component
+- Remove `odsComponentStageRolloutOpenShiftDeployment(context)` from your `Jenkinsfile`. We only want to build a docker image, not run it outside the pipeline.
+- Replace `FROM alpine:latest` with `FROM opendevstackorg/ods-jenkins-agent-base-ubi8:latest` in your `Dockerfile`.
```

Comment:
I think it would help to explain that you need to prefix with the internal registry, or adjust the YAML of the BuildConfig ... I think this is a common issue for people new to OpenShift ...

Reply:
I rephrased it to: 

_In your `Dockerfile`, replace `FROM alpine:latest` with the ods-jenkins-agent-base image that is available in the OpenDevStack namespace of your cluster, e.g. `FROM docker-registry.default.svc:5000/ods/jenkins-agent-base:latest`._

Do you think that is more clear now?